woningwaardering/stelsels/zelfstandige_woonruimten/buitenruimten/buitenruimten.py

```diff
@@ -84,6 +85,16 @@ def bereken(
                             f"Ruimte {ruimte.naam} ({ruimte.id}) is een met {ruimte.gedeeld_met_aantal_eenheden} gedeelde buitenruimte met een (h, l, b) kleiner dan (2, 1.5, 1.5) en wordt daarom niet gewaardeerd."
                         )
                         continue
+                    # Parkeerplaatsen worden alleen gewaardeerd als privé-buitenruimten
```

Comment:
Voor de helderheid zou ik hier de tekst uit het beleidsboek gebruiken:
```suggestion
                    # Gedeelde buitenruimte die als parkeerplek bedoeld is,
                    # wordt gewaardeerd volgens rubriek 10.
```